tla/tlc.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+#
+# Python TLC wrapper script for the CCF project
+# Goals:
+# - No dependencies, no venv, no pip install
+# - Set sensible defaults with an eye on performance
+# - Capture useful switches for CI, debugging
+# - Expose specification configuration through CLI
+# - Provide a useful --help, and basic sanity checks
+
+import os
+import argparse
+import pprint
+import pathlib
+
+DEFAULT_JVM_ARGS = [
+    "-XX:+UseParallelGC",  # Use parallel garbage collection, for performance
+]
+
+
+DEFAULT_CLASSPATH_ARGS = ["-cp", "tla2tools.jar:CommunityModules-deps.jar"]
+
+
+def cli():
+    parser = argparse.ArgumentParser(
+        description="TLC model checker wrapper for the CCF project"
+    )
+
+    # Common options for all commands
+    parser.add_argument(
+        "-v",
+        action="store_true",
+        help="Print out command and environment before running",
+    )
+    # Changes to TLC defaults
+    parser.add_argument(
+        "--disable-cdot", action="store_true", help="Disable \\cdot support"
```

Comment:
If we get around to removing `\cdot` from `ccfraft.tla` then maybe we will switch from enabled by default to disabled by default but I'll ok with this as is [related: https://github.com/microsoft/CCF/issues/6490#issuecomment-2363813038]

Reply:
Doesn't the fact that this script is mentioned [doc/architecture/raft_tla.rst](https://github.com/microsoft/CCF/pull/6513/files#diff-8c975bc4689cc3b98439557144ffb1698dd4d95db8fdd3c353d9278fb50e319c) make it a "public" script?